### DISCUSSION TITLE
Finding a way to automate the cog browser

### DISCUSSION THREAD

**Discussion by ntcmp2u** [Igalia/cog#491] [Q&A] [answered]
I would like to write a script to let the cog browser visit a series of websites one by one. Traditional web browsers provide webdriver to allow others to use selenium to automate them. Does the cog browser provide something like this?

**Comment by philn** [comment]
That was added in Cog 0.12. See https://github.com/Igalia/cog/pull/305

**Comment by philn** [comment, marked as answer]
https://igalia.github.io/cog/webdriver.html
